package.json

```diff
@@ -159,5 +159,8 @@
       "rootDir": "src"
     }
   },
-  "//": "~~ Generated by projen. To modify, edit .projenrc.ts and run \"npx projen\"."
+  "//": "~~ Generated by projen. To modify, edit .projenrc.ts and run \"npx projen\".",
+  "dependencies": {
+    "@cedar-policy/cedar-wasm": "^3.2.3"
+  }
```

Comment:
In Projen projects dependencies should not be directly added to the `package.json` file. They should be added to the .projenrc.ts file and then projen will deal with it during `npx projen default` or `npx projen build`  or `npx projen compile` etc. commands, by means it will download the package and add the dependency to the right set of files. Please remove the dependency from here and follow the right path :wink:  See here for more infos:

- https://projen.io/docs/api/projen/#dependencies-

- example in another cdk labs project: https://github.com/cdklabs/cdk-monitoring-constructs/blob/23caa554af7db83d63d4b22c08f2403a76328952/.projenrc.ts#L82